config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml

```diff
@@ -72,6 +72,7 @@ periodics:
       - --env=KUBE_DNS_MEMORY_LIMIT=300Mi
       - --extract=ci/latest
       - --gcp-nodes=5000
+      - --gcp-node-size=e2-standard-2
```

Comment:
As discussed offline - we need to ensure that we have enough quota for these jobs.